aws/src/test/java/org/apache/iceberg/aws/s3/signer/TestS3V4RestSignerClient.java

```diff
@@ -133,6 +150,18 @@ public static Stream<Arguments> validOAuth2Properties() {
                 "token",
                 OAuth2Properties.CREDENTIAL,
                 "user:12345"),
+            "token"),
+        // Custom scope
+        Arguments.of(
+            Map.of(
+                S3_SIGNER_URI,
+                "https://signer.com",
+                AuthProperties.AUTH_TYPE,
+                AuthProperties.AUTH_TYPE_OAUTH2,
+                OAuth2Properties.CREDENTIAL,
+                "user:12345",
+                OAuth2Properties.SCOPE,
+                "custom"),
```

Comment:
can we have a test that verifies that the default scope is `sign` when no custom scope is defined?